test/DebugInfo/irgen_void_tuple.sil

```diff
@@ -0,0 +1,26 @@
+// RUN: %target-swiftc_driver -Xfrontend -disable-debugger-shadow-copies -g -emit-ir %s
```

Comment:
Tests that just check for no crash easily bitrot, because they still keep passing even if someone replaced `%target-swiftc_driver` to expand to `/bin/true`, so it's always better to FileCheck something.

In this case `CHECK-NOT: debug_value` might do the trick.